lib/ui/pages/score/score_page.dart

```diff
@@ -125,188 +142,160 @@ class _ScoreViewerPageState extends State<ScoreViewerPage> with TickerProviderSt
   @override
   void dispose() {
     _scrollController.dispose();
-    _tabController.dispose();
+    _tabController?.dispose();
     super.dispose();
   }
 
-  _onPopupMenuSelect(int value) async {
-    switch (value) {
-      case 0:
-        _addScoreRankTask();
-        break;
-      case 1:
-        _addSearchCourseTypeTask();
-        break;
-      default:
-        break;
-    }
-  }
-
   @override
-  Widget build(BuildContext context) {
-    return DefaultTabController(
-      length: tabLabelList.length,
-      child: Scaffold(
-        appBar: AppBar(
-          title: Text(R.current.searchScore),
-          actions: [
-            if (courseScoreList.isNotEmpty)
-              PopupMenuButton<int>(
-                onSelected: (result) {
-                  setState(() {
-                    _onPopupMenuSelect(result);
-                  });
-                },
-                itemBuilder: (BuildContext context) => [
-                  PopupMenuItem(
-                    value: 0,
-                    child: Text(R.current.refresh),
-                  ),
-                  PopupMenuItem(
-                    value: 1,
-                    child: Text(R.current.calculationCredit),
-                  ),
-                ],
-              ),
-          ],
-          bottom: TabBar(
-            controller: _tabController,
-            labelColor: AppColors.mainColor,
-            unselectedLabelColor: Colors.white,
-            indicatorSize: TabBarIndicatorSize.label,
-//      labelPadding: EdgeInsets.symmetric(horizontal: 8),
-            indicator: const BoxDecoration(
-              borderRadius: BorderRadius.only(
-                topLeft: Radius.circular(10),
-                topRight: Radius.circular(10),
+  Widget build(BuildContext context) => DefaultTabController(
+        length: tabLabelList.length,
+        child: Scaffold(
+          appBar: AppBar(
+            title: Text(R.current.searchScore),
+            actions: [
+              if (courseScoreList.isNotEmpty)
+                IconButton(
+                  onPressed: _addScoreRankTask,
+                  icon: const Icon(Icons.refresh),
+                ),
+              if (courseScoreList.isNotEmpty)
+                IconButton(
+                  onPressed: _addSearchCourseTypeTask,
+                  icon: const Icon(Icons.calculate_rounded),
+                ),
```

Comment:
I think it would be a good idea to wrap the icons with a [Tooltip](https://api.flutter.dev/flutter/material/Tooltip-class.html) and show the text that was in the menu selection previously.
How do you think?

Reply:
Nice.
addressed in [616913c](https://github.com/NEO-TAT/tat_flutter/pull/157/commits/616913cacf2d6603439af8873edbb09b5f688f4d).